### PR TITLE
Users/eknuviad

### PR DESCRIPTION
**Pull request by eknuviad**
Contains the Faceted search and resolved conflicts from modifying same js files.

### CODE REVIEW

File: sps-team-38/src/main/webapp/index.html [hunk 1]
```diff
@@ -49,7 +49,7 @@ <h3>Request an Volunteer Here!</h3>
                 <option>Spanish</option>
                 <option>Arabic</option>
             </datalist> <br>
-            <label for="meet_link">Paste a link to a Google Meet chat for your interview:</label>
+            <label for="meet_link">Paste a link to a Google Meet session for your volunteer(s) to join:</label>
```
Comment (alexschles) [inline sps-team-38/src/main/webapp/index.html:52]: No big deal just kind of confused why the change wasn't already there after my PR was merged

File: sps-team-38/src/main/webapp/script.js [hunk 2]
```diff
@@ -53,3 +53,51 @@ function addInput(divName){
...
+const list = document.querySelector('#form-list ul');
+  // Load existing forms in database
+function loadForms(){
+    fetch('/list-forms').then(response => response.json()).then((forms) => {
+    forms.forEach((form) => {
+        formListElement.appendChild(renderForm(form));
+    })
+    });
+}
+// ................remove this section, just for reference....................
```
Comment (alexschles) [inline sps-team-38/src/main/webapp/script.js]: Is "this section" referring to renderForm? If so, why is it part of the PR? If not, and if this is relating to something already removed, I think we should remove this comment to avoid confusion down the line.

File: sps-team-38/src/main/webapp/script.js [hunk 3]
```diff
@@ -53,26 +53,44 @@ function addInput(divName){
...
 const list = document.querySelector('#form-list ul');
+
   // Load existing forms in database
 function loadForms(){
-    fetch('/list-forms').then(response => response.json()).then((forms) => {
+    // fetch('/list-forms').then(response => response.json()).then((forms) => {
+    // forms.forEach((form) => {
+    //     formListElement.appendChild(renderForm(form));
+    // })
+    // });
```
Comment (alexschles) [inline sps-team-38/src/main/webapp/script.js]: If this isn't being used I don't think it should be in the PR either to avoid confusion down the line (just think that's best practice- my PA from Spring session always stopped me when I was going to do this)

**Review by alexschles — APPROVED** [review]
Can't say I fully understand the whole flow of what's going on here, but maybe you can show us at the team meeting tomorrow. I trust that whatever is going on works, so I'll approve the PR and feel free to merge after reviewing/adjusting based on my comments.

File: sps-team-38/src/main/webapp/index.html (hunk 1 above)
Comment (eknuviad) [reply sps-team-38/src/main/webapp/index.html:52]: in resolving the conflict on my branch it probably got lost.

File: sps-team-38/src/main/webapp/script.js (hunk 2 above)
Comment (eknuviad) [reply sps-team-38/src/main/webapp/script.js]: Right this was a reminder to my future self to make changes to this section. Will remove this. Thanks

File: sps-team-38/src/main/webapp/script.js (hunk 3 above)
Comment (eknuviad) [reply sps-team-38/src/main/webapp/script.js]: Got it. Will remove.
